src/downshift.js

```diff
@@ -656,30 +658,42 @@ class Downshift extends Component {
       )
     }
     this.inputId = firstDefined(this.inputId, rest.id, `${this.id}-input`)
-    let onChangeKey
-    /* istanbul ignore next (preact) */
-    if (preval`module.exports = process.env.BUILD_PREACT === 'true'`) {
-      onChangeKey = 'onInput'
-      /* istanbul ignore next (react-native) */
+    let eventHandlers
+    /* istanbul ignore next*/
+    if (rest.disabled) {
+      eventHandlers = {}
+    } else if (preval`module.exports = process.env.BUILD_PREACT === 'true'`) {
+      eventHandlers = {
+        onInput: composeEventHandlers(
+          onChange,
+          onInput,
+          this.input_handleChange,
+        ),
+        onKeyDown: composeEventHandlers(onKeyDown, this.input_handleKeyDown),
+        onBlur: composeEventHandlers(onBlur, this.input_handleBlur),
+      }
     } else if (
       preval`module.exports = process.env.BUILD_REACT_NATIVE === 'true'`
     ) {
-      onChangeKey = 'onChangeText'
+      eventHandlers = {
```

Comment:
What's the rationale behind removing `onBlur` from the event handlers that we use with React Native?